powertools-e2e-tests/src/test/java/software/amazon/lambda/powertools/testutils/JavaRuntime.java

```diff
@@ -0,0 +1,37 @@
+package software.amazon.lambda.powertools.testutils;
+
+import software.amazon.awscdk.services.lambda.Runtime;
+
+public enum JavaRuntime {
+    JAVA8("java8", Runtime.JAVA_8, "1.8"),
```

Comment:
Can we use Java8 here ? The source/SDK version are forced in the E2E test parent pom. Not sure if that impacts here.

Reply:
This is dynamically changed, thanks to the JAVA_VERSION env variable: 
See [here](https://github.com/jeromevdl/aws-lambda-powertools-java/blob/94252a4a7dfec6c7bbf8274c21662b5aa3f865ef/powertools-e2e-tests/src/test/java/software/amazon/lambda/powertools/testutils/Infrastructure.java#L205) and [here](https://github.com/jeromevdl/aws-lambda-powertools-java/blob/94252a4a7dfec6c7bbf8274c21662b5aa3f865ef/powertools-e2e-tests/src/test/java/software/amazon/lambda/powertools/testutils/Infrastructure.java#L237) and [here](https://github.com/jeromevdl/aws-lambda-powertools-java/blob/94252a4a7dfec6c7bbf8274c21662b5aa3f865ef/powertools-e2e-tests/src/test/java/software/amazon/lambda/powertools/testutils/Infrastructure.java#L141)